infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/send/logreader/StreamsLogEntryReader.java

```diff
@@ -185,6 +185,10 @@ public void setGlobalBaseSnapshot(long snapshot, long ackTimestamp) {
 
     @Override
     public LogReplicationEntryMsg read(UUID logEntryRequestId) throws TrimmedException {
+        log.info("Last Opaque Entry = {}", lastOpaqueEntry);
```

Comment:
to sure if this is one of the statements you aim to remove, but just in case not, I believe this log statement could be noisy

Reply:
Looking at the whole class, I'm guessing all these will be removed, is that right?

Reply:
yes, that's right